src/ucp/wireup/address.c

```diff
@@ -654,7 +654,8 @@ ucp_address_pack_iface_attr_v1(ucp_worker_h worker, void *ptr,
     packed->overhead       = iface_attr->overhead;
     packed->bandwidth      = ucp_tl_iface_bandwidth(worker->context,
                                                     &iface_attr->bandwidth);
-    packed->lat_ovh        = iface_attr->latency.c;
+    packed->lat_ovh        = ucp_tl_iface_latency(worker->context,
+                                                  &iface_attr->latency);
```

Comment:
looks like it is not wire compatible

Reply:
can you explain?

Reply:
@brminich right. But there is a bug here. Do you think we can do any better than just keeping it there?

Reply:
@brminich why do you think it's not wire-compatible? from remote side perspective it always treats this value from address as latency number

Reply:
pls revert this to preserve old v1 logic